velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -183,6 +183,64 @@ TEST_F(ArithmeticTest, Divide) {
   EXPECT_TRUE(std::isnan(divide(kInf, -kInf).value_or(0)));
 }
 
+TEST_F(ArithmeticTest, acosh) {
```

Comment:
nit: wondering if it makes sense to add only one case each to these tests that has a valid value and a valid result

Reply:
Corner cases should enough, other cases may have precision problem on different compiler.